docker-compose.yml

```diff
@@ -1,93 +1,78 @@
 version: '3'
 services:
-  node-0:
-    image: cennznet/cennznet:0.9.24-beta-639f5f36
+  node_alice:
```

Comment:
This should be simplified to a single node.
The reasoning being the functionality of the API under test should not change with the number of nodes on the network.
- `volumes` and  mounting stuff is not required here since nothing should be persisted between CI test runs
- I don't think a UI server needs to be spun up as part of this CI pipeline

This file should be really simple with two services a CENNZnet node container and an API container to run the e2e tests
```yaml
services:
  cennznet-node:
    image: cennznet/cennznet:rc1
    command:
      - -- dev
    ports:
      - "9933:9933"
      - "9944:9944"
  cennznet-api:
   # ...
```

Reply:
Yes, The UI together with multiple nodes was to demo isolated test network in docker, but definitely not need in CI,    will simplify it